website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -125,6 +124,8 @@ The variables must be set with the proper values in the `terraform.tfvavrs` file
 
 :::note Optional Cumulus Values
 
+aws_profile should be set if using an aws_profile other than `default` for Cumulus' deployment.
```

Comment:
This isn't very clear. I guess I am not sure what you are trying to convey here.